docs/book/src/user/quick-start.md

```diff
@@ -158,6 +158,23 @@ Test to ensure the version you installed is up-to-date:
 clusterctl version
 ```
 
+{{#/tab }}
+{{#tab windows}}
+
+#### Install clusterctl binary with curl on Windows using PowerShell
+Go to the working directory where you want clusterctl downloaded.
+
+Download the latest release; on windows, type:
+```
+curl {{#releaselink gomodule:"https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.1.1/clusterctl-windows-amd64.exe"}} -o clusterctl.exe
```

Comment:
```suggestion
curl {{#releaselink gomodule:"sigs.k8s.io/cluster-api" asset:"clusterctl-windows-amd64.exe" version:"1.1.x"}} -o clusterctl.exe
```

This will fix the pipeline errors you are seeing on the pipeline.

Reply:
Surely will commit this change, once everything is clear like what link to be added and all for setting env variables.
That will help to make all changes in one commit ig which will be better. :v: